tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -17,21 +18,26 @@ final class BuildCommand extends CommandBase {
   }) {
     builds = runnableBuilds(environment, configs);
     debugCheckBuilds(builds);
-    // Add options here that are common to all queries.
-    argParser.addOption(
-      configFlag,
-      abbr: 'c',
-      defaultsTo: 'host_debug',
-      help: 'Specify the build config to use',
-      allowed: <String>[
-        for (final Build config in runnableBuilds(environment, configs))
-          config.name,
-      ],
-      allowedHelp: <String, String>{
-        for (final Build config in runnableBuilds(environment, configs))
-          config.name: config.gn.join(' '),
-      },
-    );
+    argParser
+      ..addOption(
+        configFlag,
+        abbr: 'c',
+        defaultsTo: 'host_debug',
+        help: 'Specify the build config to use',
+        allowed: <String>[
+          for (final Build config in runnableBuilds(environment, configs))
+            config.name,
+        ],
+        allowedHelp: <String, String>{
+          for (final Build config in runnableBuilds(environment, configs))
+            config.name: config.gn.join(' '),
+        },
+      )
+      ..addFlag(
+        fetchFlag,
+        defaultsTo: null,
+        help: 'Download dependencies',
+      );
```

Comment:
Please remove this flag. 

We don't need N ways of requesting something happens. Also, if we go down this path then every command will start adding a bunch of flags which really just trigger other commands.

Reply:
Capturing our conversation offline: in the future, `et build` will automagically download dependencies only when needed. Assuming the dependency logic is correct, we don't need an option to disable this.

I've removed the `--fetch` and `--no-fetch` options accordingly.

Reply:
Thanks!